subsys/bluetooth/controller/Kconfig

```diff
@@ -516,7 +516,7 @@ config BT_CTLR_PHY_CODED
 
 config BT_CTLR_ZLI
 	bool "Use Zero Latency IRQs"
-	depends on ZERO_LATENCY_IRQS
+	depends on ZERO_LATENCY_IRQS && BT_LL_SW_SPLIT
```

Comment:
Then I would move the whole option into the if BT_LL_SW_SPLIT

Reply:
Use of ZLI is under advanced features menu, moving it out will make it visible in the top Bluetooth menu. Hence, continue to keep it with depends on.

Reply:
ok